RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch prevents failing stateful tests from suggesting that you add a 
+@reproduce_failure call, which are not applicable in that case.
```

Comment:
Three things here:

1. This file needs to be moved to `hypothesis-python/RELEASE.rst`, so that our build script knows which version of Hypothesis to release (i.e., not the Ruby one :wink:)
2. Hyperlinks to functions like `@reproduce_failure` are nice, though the Sphinx syntax for them isn't particularly obvious.  Example below.
3. We like to credit contributors in the changelog for their release - this is optional, but if you're comfortable with the wording below that would be great.  Substitute in whatever name or handle you prefer both here and in the authors list, obviously!

```
RELEASE_TYPE: patch

This patch stops failing stateful tests from suggesting that you add a 
:func:`@reproduce_failure <hypothesis.reproduce_failure>` decorator, 
which is a leaking implementation detail and not applicable here.

Thanks to El Awbery for this patch!
```

Reply:
Made the suggested changes. Not sure whether the link works? I don't have the Sphinx extension to check it...gotta get on a plane soon, so I will fix it later if it didn't work.